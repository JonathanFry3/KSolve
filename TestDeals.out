STOCK    : KC 2D TS AS 6D 3D TC AC AH 3H 5H 2C JD 6H 9D 8D KS 3S 7D 5S 8H 6S 2H QD 
WASTE    : 
TABLEAU 1: 4D 
TABLEAU 2: AD -4C
TABLEAU 3: 5D -QS-4H
TABLEAU 4: 6C -5C-8C-QH
TABLEAU 5: 4S -9C-QC-TH-2S
TABLEAU 6: KH -7C-8S-9H-9S-7S
TABLEAU 7: JH -JS-KD-TD-7H-3C-JC
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 104

Minimal solution in 122 moves.
Took 5.07856 sec. 1.03 million unique states.
STOCK    : 3D 5H TC 4C TS KD KC 2D 5S 4D AS 2C TD 8H TH 7H JD 8D 8S 9D 5C QS KH 3C 
WASTE    : 
TABLEAU 1: AH 
TABLEAU 2: 4H -7D
TABLEAU 3: QH -AC-2H
TABLEAU 4: JC -6D-8C-9S
TABLEAU 5: 6H -7S-KS-AD-2S
TABLEAU 6: 6S -3S-6C-QC-9H-JH
TABLEAU 7: 7C -9C-3H-QD-JS-5D-4S
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 104

Minimal solution in 130 moves.
Took 6.03 sec. 1.88 million unique states.
STOCK    : 8H 6H QH 3C KS 8D 9D 4D 9C QC 2H 7S KC JC 7H 2D 2C 5C JD KD 5D QD 7C TC 
WASTE    : 
TABLEAU 1: 3S 
TABLEAU 2: 3H -TS
TABLEAU 3: TH -3D-KH
TABLEAU 4: TD -8S-9H-7D
TABLEAU 5: 6C -AS-5S-8C-QS
TABLEAU 6: AD -JH-9S-4C-4H-6D
TABLEAU 7: 6S -2S-AC-5H-JS-4S-AH
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 102

Minimal solution in 191 moves.
Took 0.784 sec. 0.136 million unique states.
STOCK    : 4D AS 9C TH 3D TC JS 6D 2D TS 5D 3H 2C KD AD 7H JD JC QS AC 5S 2H 8C AH 
WASTE    : 
TABLEAU 1: 4C 
TABLEAU 2: JH -2S
TABLEAU 3: 9D -6S-KH
TABLEAU 4: QD -KC-7D-QH
TABLEAU 5: 6C -8S-4S-3C-KS
TABLEAU 6: 9H -5C-8H-8D-3S-7C
TABLEAU 7: 7S -5H-9S-6H-4H-QC-TD
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 103

Minimal solution in 132 moves.
Took 6.52 sec. 1.48 million unique states.
STOCK    : TC TS AD 4D QS 3H 3S JH TD KC 8H 8S 4S 4H KS 6D 8D 7C JD 9D 2H QD 3D AS 
WASTE    : 
TABLEAU 1: 5C 
TABLEAU 2: 9H -7H
TABLEAU 3: AC -9S-2D
TABLEAU 4: 8C -5S-QH-5D
TABLEAU 5: 7D -3C-JS-2C-QC
TABLEAU 6: 6S -JC-9C-KD-7S-5H
TABLEAU 7: KH -TH-6H-6C-2S-AH-4C
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 104

Unknown.
Took 40.1 sec. 10 million unique states.
