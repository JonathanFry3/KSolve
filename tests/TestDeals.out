WASTE    : 
TABLEAU 1: 4D 
TABLEAU 2: AD -4C
TABLEAU 3: 5D -QS-4H
TABLEAU 4: 6C -5C-8C-QH
TABLEAU 5: 4S -9C-QC-TH-2S
TABLEAU 6: KH -7C-8S-9H-9S-7S
TABLEAU 7: JH -JS-KD-TD-7H-3C-JC
STOCK    : -KC-2D-TS-AS-6D-3D-TC-AC-AH-3H-5H-2C-JD-6H-9D-8D-KS-3S-7D-5S-8H-6S-2H-QD
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 83

Minimal solution in 101 moves + 21 flips in 1 pass.
Took 0.4519 sec. 1.077 million moves generated.
WASTE    : 
TABLEAU 1: AH 
TABLEAU 2: 4H -7D
TABLEAU 3: QH -AC-2H
TABLEAU 4: JC -6D-8C-9S
TABLEAU 5: 6H -7S-KS-AD-2S
TABLEAU 6: 6S -3S-6C-QC-9H-JH
TABLEAU 7: 7C -9C-3H-QD-JS-5D-4S
STOCK    : -3D-5H-TC-4C-TS-KD-KC-2D-5S-4D-AS-2C-TD-8H-TH-7H-JD-8D-8S-9D-5C-QS-KH-3C
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 83

Minimal solution in 109 moves + 21 flips in 1 pass.
Took 0.6048 sec. 2.259 million moves generated.
WASTE    : 
TABLEAU 1: 3S 
TABLEAU 2: 3H -TS
TABLEAU 3: TH -3D-KH
TABLEAU 4: TD -8S-9H-7D
TABLEAU 5: 6C -AS-5S-8C-QS
TABLEAU 6: AD -JH-9S-4C-4H-6D
TABLEAU 7: 6S -2S-AC-5H-JS-4S-AH
STOCK    : -8H-6H-QH-3C-KS-8D-9D-4D-9C-QC-2H-7S-KC-JC-7H-2D-2C-5C-JD-KD-5D-QD-7C-TC
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 81

Minimal solution in 170 moves + 21 flips in 5 passes.
Took 0.06094 sec. 0.134 million moves generated.
WASTE    : 
TABLEAU 1: 4C 
TABLEAU 2: JH -2S
TABLEAU 3: 9D -6S-KH
TABLEAU 4: QD -KC-7D-QH
TABLEAU 5: 6C -8S-4S-3C-KS
TABLEAU 6: 9H -5C-8H-8D-3S-7C
TABLEAU 7: 7S -5H-9S-6H-4H-QC-TD
STOCK    : -4D-AS-9C-TH-3D-TC-JS-6D-2D-TS-5D-3H-2C-KD-AD-7H-JD-JC-QS-AC-5S-2H-8C-AH
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 82

Minimal solution in 111 moves + 21 flips in 2 passes.
Took 0.541 sec. 1.563 million moves generated.
WASTE    : 
TABLEAU 1: 5C 
TABLEAU 2: 9H -7H
TABLEAU 3: AC -9S-2D
TABLEAU 4: 8C -5S-QH-5D
TABLEAU 5: 7D -3C-JS-2C-QC
TABLEAU 6: 6S -JC-9C-KD-7S-5H
TABLEAU 7: KH -TH-6H-6C-2S-AH-4C
STOCK    : -TC-TS-AD-4D-QS-3H-3S-JH-TD-KC-8H-8S-4S-4H-KS-6D-8D-7C-JD-9D-2H-QD-3D-AS
CLUBS    : 
DIAMONDS : 
SPADES   : 
HEARTS   : 
Minimum Moves Needed: 83

Minimal solution in 115 moves + 21 flips in 2 passes.
Took 12.58 sec. 32.39 million moves generated.
